documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -130,23 +276,45 @@ you need to enable the Groovy plugin by adding the following snippet to your Ela
 script.engine.groovy.inline.search: on
 ----
 
-==== Limitations
+==== Projections
 
-Many features are not implemented yet: this preview is not very useful yet!
-On the other hand, implementing some of these is not very hard: this might be the perfect time to join the project.
+All fields are stored by Elasticsearch in the JSON document it indexes,
+there is no specific need to mark fields as stored.
+It is a good practice however to do it, it will make your code more portable
```

Comment:
OK then I don't understand how it works, in ES's case, adding Store.YES is a noop, right? So why is that a problem for a user to add Store.YES on the fields it wants to project?